docs/application/web/guides/connectivity/download.md

```diff
@@ -85,25 +85,45 @@ To provide the user access to Internet resources, you must learn how to manage d
 
    The `start()` method returns a unique identifier for the download operation.
 
-5. During the download:
+   You can check the status of the download operation by calling the getState() method with the download ID:
+
+      ```
+      tizen.download.getState(downloadId);
+      ```
+
+5. During the download operation, you can modify the download state.
+
+    >**Note**
+    >
+    >Download operation can become completed almost instantaneously, depending on download size and connection speed.
```

Comment:
@apietraszek 
Kindly change: 
 >Download operation can become completed almost instantaneously, depending on download size and connection speed.

to 

>The download operation gets completed almost instantaneously, depending on download size and connection speed.

Reply:
Done